backend/app/api/endpoints/health.py

```
@@ -35,12 +35,24 @@ def health_check(db: Session = Depends(get_db)):
+            admin_password_changed = (
+                admin_user.password_hash != DEFAULT_ADMIN_Password_HASH
+            )
+
         return {
             "status": "healthy",
             "database": "connected",
             "users_initialized": user_count > 0,
             "user_count": user_count,
             "shutting_down": shutdown_manager.is_shutting_down,
+            "admin_password_changed": admin_password_changed,
         }
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)